extensions/blocks/podcast-player/style.scss

```diff
@@ -160,7 +161,8 @@ $player-float-background: $light-gray-200;
 		max-height: 105px; //IE11 fallback
 	}
 
-	.has-secondary { // custom color.
+	.has-secondary {
+		// custom color.
```

Comment:
This applies to the line above. Let's remove it or add it above the line if we need to keep it

Reply:
Updated in https://github.com/Automattic/jetpack/pull/15281/commits/bdbed525ff616f1e87c598a29b9b51aa732badf3